runtime/handler.go

```diff
@@ -60,7 +60,13 @@ func ForwardResponseStream(ctx context.Context, w http.ResponseWriter, req *http
 }
 
 // ForwardResponseMessage forwards the message "resp" from gRPC server to REST client.
-func ForwardResponseMessage(ctx context.Context, w http.ResponseWriter, req *http.Request, resp proto.Message) {
+func ForwardResponseMessage(ctx context.Context, w http.ResponseWriter, req *http.Request, resp proto.Message, opts ...func(context.Context, http.ResponseWriter, proto.Message) error) {
```

Comment:
how do you create an instance of this then? I originally did this, but is it legal to do:

```go
someOption = runtime.ForwardResponseOption(ctx context.Context, w http.ResponseWriter, message proto.Message) { ... }
```

I was not sure how to do this, to be honest.